index.js

```diff
@@ -47,4 +52,53 @@ app.post('/command', (req, res) => {
   res.send('Connection to localtunnel verified. Ready for your next adventure?')
 })
 
+function sendMessageToSlackResponseURL(responseURL, JSONmessage) {
+  const postOptions = {
+    uri: responseURL,
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json'
+    },
+    json: JSONmessage
+  }
+  request(postOptions, (err, res, body) => {
+    if (err) {
+      console.error(err)
+      res.sendStatus(500)
```

Comment:
This `res` object is the response _received_ by `request`, not a `res` that `express` uses to _send_ responses.